dace/__init__.py

```diff
@@ -15,6 +15,8 @@
 from .memlet import Memlet
 from .symbolic import symbol
 
+import dace.library
```

Comment:
Is this necessary? it slows down the import time of dace

Reply:
Not sure, but see the commit before "Small Fixes", it was failing CI. 

Also; this fixes long standing static checker errors in PyCharm

Reply:
well, something is adding 5 more minutes to the build time... probably not this, unsure